modelmaterials_gl4/001_units_s3o_assimp.lua

```diff
@@ -394,6 +419,8 @@ for id = 1, #UnitDefs do
 				unitMaterials[id] = {"unitsNormalMapArmTanks", NORMALTEX = normalTex, TEXW1 = wreckAtlas[1], TEXW2 = wreckAtlas[2], NORMALTEX2 = wreckAtlas[3]}
 			elseif facName == "cor" then
 				unitMaterials[id] = {"unitsNormalMapCorTanks", NORMALTEX = normalTex, TEXW1 = wreckAtlas[1], TEXW2 = wreckAtlas[2], NORMALTEX2 = wreckAtlas[3]}
+			elseif facName == "ion" then -- temporary prefix for legion during transition from cor/arm to legion atlas
+				unitMaterials[id] = {"unitsNormalMapLegTanks", NORMALTEX = normalTex, TEXW1 = wreckAtlas[1], TEXW2 = wreckAtlas[2], NORMALTEX2 = wreckAtlas[3]}
 			end
 		else
```

Comment:
these if statements should also be making use of the new atlas parameter right?

Reply:
updated facName to be either the first 3 letters from the atlas customparam, faction customparam, or unit name. swaped this back to check for the 3 letters "leg"